### PR TITLE
feat: #712 hide fields Guidelines

### PR DESCRIPTION
**Pull request by Anok666**

### CODE REVIEW

**Review by tomaszandel — APPROVED** [review]

**Comment by Marcelinna** [comment]
image

**Comment by Marcelinna** [comment]
image

**Review by Marcelinna — APPROVED** [review]

**Review by Marcelinna — DISMISSED** [review]
If we dont have value, header should be not visible too
image

**Comment by Marcelinna** [comment]
image
Domain the same situation as above

**Comment by Marcelinna** [comment]
Despite I have value for Name Type*
I don't see the value in SS
image
image

**Comment by Marcelinna** [comment]
I have Name Type: Personal, despite value is organizational
image
image

**Comment by eldenkat** [comment]
Additional changes required

- [ ]  Remove "eosc." from the Provider information -> Provider:  (now: eosc.geant/should be: geant) [**warning**: in case it is possible act alternatively: Required fix is that the Provider name should print as in the Service preview in the MP
image
Alternative:
https://cyfronet-fid.slack.com/files/U041F9A92UV/F05KWRNACAX/zrzut_ekranu_2023-08-02_105733.png
- [ ] Provider Information panel: **Remove 'Provider'+ "providerId": "string"** 
- [ ] Provider Information panel: **Remove 'Catalogue' + Catalogue ID (this is not required)**
- [ ] Provider Information panel: **change title to 'Creator information'** and **flow Creator data over all three columns**
- [ ] Do not print attribute field names when data is null (e.g. currently printing 'EOSC Integration Options = -'

**Comment by eldenkat** [comment]
Overall it looks ok, but the spacing doesn't match:

Zrzut ekranu 2023-08-07 105907

**Comment by eldenkat** [comment]
Domain is empty, should't be displayed.
Zrzut ekranu 2023-08-08 072507

**Review by eldenkat — APPROVED** [review]

File: ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts [hunk 1]
```diff
@@ -33,6 +36,11 @@ export class GuidelineDetailPageComponent implements OnInit {
     private _router: Router
   ) {}
 
+  parseCreators(jsonDeserialized: string | undefined): Partial<ICreators>[] {
+    if (jsonDeserialized) console.log(JSON.parse(jsonDeserialized));
```
Comment (michal-szostak) [inline ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts:40]: `if`s without brackets can lead to issues - I would add them.

**Review by michal-szostak — APPROVED** [review]

File: ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts (hunk 1 above)
Comment (Anok666) [reply ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts:40]: true

Comment (Anok666) [reply ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts]: done

**Review by michal-szostak — APPROVED** [review]

**Pull request merged**